op-chain-ops/script/script.go

```diff
@@ -297,7 +297,7 @@ func NewHost(
 
 // AllowCheatcodes allows the given address to utilize the cheatcodes and logging precompiles
 func (h *Host) AllowCheatcodes(addr common.Address) {
-	h.log.Debug("Allowing cheatcodes", "address", addr, "label", h.labels[addr])
+	h.log.Trace("Allowing cheatcodes", "address", addr, "label", h.labels[addr])
```

Comment:
A few questions:
- Are we using Trace-level logging anywhere?
- If we almost never do, might as well remove this log if it doesn't serve a good-level of info?
- If the noise happens very incidentally, say, when running op-deployer (just guessing on the basis of the docs I read recently), not sure if it's any harm to keep the log as it is because it might help with any troubleshooting during other relevant situations.

Reply:
We use trace logging in a bunch of places. The noise happens every time we run op-deployer in the tests, which is quite a lot. This log alone produces several thousand lines of log spam, since we allow cheatcodes for almost every predeploy.